src/Diagnostics.DataProviders/DataProviders/ChangeAnalysisDataProvider.cs

```diff
@@ -199,5 +200,17 @@ private List<string> GetHostNamesFromTable(DataTable kustoTable)
 
             return hostnames;
         }
+
+        /// <summary>
+        /// Forwards the request to Change Analysis Client
+        /// </summary>
+        /// <param name="requestUri">Request URI</param>
+        /// <param name="postBody">Post body</param>
+        /// <param name="method">HTTP Method.</param>
+        /// <returns>JSON string</returns>
+        public async Task<string> ForwardRequestToClient(string requestUri, object postBody = null, HttpMethod method = null)
```

Comment:
nitpick : Can the name be little bit more obvious to discover? like InvokeChangeAnalysisRequest or something like that?